.gitignore

```diff
@@ -7,6 +7,14 @@
 *.rsuser
 *.suo
 *.user
+/eval
```

Comment:
The current *.gitignore* should have excluded almost everything unnecessary. It's safe to delete files that are not in the list.

Reply:
If it wasn't clear enough: could you please revert this file?

Reply:
I prefer that. How could I do that?

Reply:
`git checkout master -- .gitignore`
`git clean -d -n`

Reply:
`git clean -d -n`   
This command twice with -n and without -n ?

Reply:
Up to you. `-n` is short for `--dry-run` and prevents you from accidentally removing files.